pages/serverless-containers/reference-content/migrate-to-serverless-containers.mdx

```diff
@@ -0,0 +1,119 @@
+
+## Key features
+
+Serverless Containers come packed with features designed for modern application development.
+
+- **Private Networks compatibility**: Securely connect your services to other resources within your Virtual Private Cloud.
+    - Learn [how to use Private Networks with Serverless Containers](/serverless-containers/reference-content/containers-private-networks/).
+
+- **Custom domains**: Easily map your own domain names to your services.
+    - Discover [how to add custom domains to Serverless Containers](/serverless-containers/how-to/add-a-custom-domain-to-a-container/).
+
+- **Healthchecks**: Configure readiness and liveness probes to ensure your application instances are healthy and ready to serve traffic.
+    - Get [information about healtchecks](/serverless-containers/concepts/#health-check).
+
+- **Built-in monitoring**: Gain immediate insights on your application's performance with integrated logging and metrics, without needing to set up complex monitoring agents.
+    -  Check [how to monitor your Serverless Container](/serverless-containers/how-to/monitor-container/).
+
+- **Enhanced security**: Because the platform is managed by Scaleway, it benefits from continuous security patching and best practices, reducing your operational security burden.
+    - Find out [how to secure a Serverless Container](/serverless-containers/how-to/secure-a-container/).
+
+To dive deeper into Serverless Containers, refer to the dedicated [concepts page](/serverless-containers/concepts/).
+
+## Quick deployment guide
+
+Deploying your first service to Serverless Containers is straightforward. Here’s a high-level overview of the steps.
+
+1. **Containerize your application**: If you haven't already, package your application into a container image using a Dockerfile.
+
+2. **Push your image to a registry**: Follow the [dedicated Quickstart documenation](/container-registry/quickstart/) to discover how to log in to the Scaleway Container Registry and store your images.
+
+3. **Deploy your container**: Read the [How to deploy a container](/serverless-containers/how-to/deploy-container/) page to create your container using the Scaleway console.
+
+4. **Click Deploy**: The Scaleway platform will pull your container image and run it. Within a few minutes, your Serverless Container will be publicly accessible via its HTTPS endpoint.
+
+## Best practices for migration
+
+Migrating existing applications from Kubernetes or Instances to Serverless Containers is an opportunity to modernize your architecture.
+
+### General best practices
+
+**Containerize everything**
+
+Your application must be packaged as a container image. This is the fundamental deployment unit for Serverless Containers.
+
+**Embrace statelessness**
+
+Since Serverless Containers are stateless, you cannot rely on the local filesystem for storing data that needs to persist between requests
```

Comment:
```suggestion
Since Serverless Containers are stateless, you cannot rely on the local filesystem for storing data that needs to persist between requests.
```